plasmapy/atomic/ionization_states.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+from .particle_class import Particle
+from .particle_input import particle_input
+from ..utils import AtomicError
+
+class IonizationState:
+    """
+    Describe the ionization state of a single element.
+    """
+    @particle_input(require='element', exclude=['isotope', 'ion'])
+    def __init__(self,
+                 element: Particle,
+                 distribution=None,
+                 T_e=None,  # not yet implemented!
+                 ):
+        """
+        Initialize the ionization state distribution.
+
+        Notes
+        -----
+        The method to calculate collisional ionization equilibrium has
+        not yet been implemented and will raise a `NotImplementedError`.
+
+        """
+        self._element = element
+
+        if distribution is None:
+            raise NotImplementedError(
+                "IonizationState instances are not yet able to "
+                "calculate collisional ionization equilibrium.")
+
+        if isinstance(distribution, (tuple, list)):
+            distribution = np.array(distribution)
+        elif not isinstance(distribution, np.ndarray):
+            raise TypeError
+
+        if len(distribution) != element.atomic_number + 1:
+            raise AtomicError("Incorrect number of ionization states.")
+
+        ionsum = np.sum(distribution)
+        if not np.isclose(ionsum, 1.0):
```

Comment:
add absolute and relative error tolerances and present them as default arguments for the class/method so that the user can adjust them?